File: Gissenger/GissengerFunction.py
import numpy as np

def f(x):
    #Gissenger coefficients
    mu = 0.119
    nu = 0.1
    gamma = 0.9
    #Gissenger values
    Q = x[0][0]
    D = x[1][0]
    V = x[2][0]
    #Time derivatives
    dQdt = mu*Q - V*D
    dDdt = -nu*D + V*Q
    dVdt = gamma - V + Q*D
    #Enter values into vector
    func = np.array([[dQdt],[dDdt],[dVdt]])

    return func

File: GissengerMatlab/GissengerFunction.m
function out = GissengerFunction(x)
Q = x(1);
D = x(2);
V = x(3);

m = 0.119;
nu = 0.1;
G = 0.9;

dQdt = m*Q-V*D;
dDdt = -nu*D+V*Q;
dVdt = G-V+Q*D;

out = [dQdt;dDdt;dVdt;0;0];